bootstrap.php

```diff
@@ -0,0 +1 @@
+<?php require_once __DIR__ . '/vendor/autoload.php';
```

Comment:
It lacks `@copyright` but my question is different: Do we need `bootstrap.php`? You can just point to the `autoload.php` which will reduce file overhead.

Reply:
It's just a convention. In another phpunit.xml file is the same. @Nattfarinn, @webhdx  what should we do?